src/image_tools/bake.py

```diff
@@ -159,18 +156,71 @@ def bake_command(args: Namespace, product_name: str, bakefile) -> Command:
     )
 
 
+def filter_product_version(conf, product_name: Optional[str], product_version: Optional[str],
+                           product_version_quantile: Optional[int], max_bake_runners: int) -> Optional[int]:
+    """ Filter product versions by the given product_version argument.
+        Mutates the "conf.products" array to remove version of "product_name" that don't match "product_version"
+
+        Returns the number of product versions to build.
+    """
+    result = None
```

Comment:
Why have this special (undocumented!) case?

Reply:
Yeah. Not needed if we raise an error as mentioned above.